src/test/java/uk/gov/companieshouse/mapper/email/DissolutionEmailMapperTest.java

```diff
@@ -89,6 +91,27 @@ public void mapToApplicationRejectedEmailData_mapsApplicationRejectedEmailData()
         assertEquals(applicationRejectedEmailData.getCompanyNumber(), result.getCompanyNumber());
         assertEquals(applicationRejectedEmailData.getCompanyName(), result.getCompanyName());
         assertEquals(applicationRejectedEmailData.getRejectReasons(), result.getRejectReasons());
+        assertEquals(applicationRejectedEmailData.getPaymentReference(), result.getPaymentReference());
+    }
+
+    @Test
+    public void mapToApplicationRejectedEmailData_mapsApplicationRejectedEmailDataCopy() {
```

Comment:
You won't need this test, and you can reuse the `mapToApplicationRejectedEmailData_mapsApplicationRejectedEmailData` test for different `emailAddress` parameters

Reply:
but I will then need to add additional logic to change the expected result in that test and there won't be coverage for the two parameter method.

Reply:
oh sorry yes this test should have been deleted